### PR TITLE
Cabbage boyscouting

### PR DESCRIPTION
**Pull request by ubergesundheit**
This PR updates some links and adds additional information

### CODE REVIEW

File: scripts/update-crd-reference/config.yaml [hunk 1]
```diff
@@ -268,7 +268,7 @@ source_repositories:
         hidden: true
       certconfigs.core.giantswarm.io:
         owner:
-          - https://github.com/orgs/giantswarm/teams/team-cabbage
+          - https://github.com/orgs/giantswarm/teams/team-phoenix
```
Comment (mcharriere) [inline scripts/update-crd-reference/config.yaml:271]:
```suggestion
          - https://github.com/orgs/giantswarm/teams/team-hydra
```
according to the `Ownership of components` spreadsheet

File: src/content/ui-api/management-api/crd/certconfigs.core.giantswarm.io.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ crd:
     - workloadcluster
 layout: crd
 owner:
-  - https://github.com/orgs/giantswarm/teams/team-cabbage
+  - https://github.com/orgs/giantswarm/teams/team-phoenix
```
Comment (mcharriere) [inline src/content/ui-api/management-api/crd/certconfigs.core.giantswarm.io.md:23]:
```suggestion
  - https://github.com/orgs/giantswarm/teams/team-hydra
```
according to the `Ownership of components` spreadsheet

**Review by mcharriere — COMMENTED** [review]
lgtm

**Review by mcharriere — APPROVED** [review]

**Comment by Gacko** [comment]
TIL: "Boyscouting".

**Review by Gacko — APPROVED** [review]

**Pull request merged**